src/main/java/org/dcsa/bkg/model/transferobjects/ShipmentLocationTO.java

```diff
@@ -0,0 +1,27 @@
+package org.dcsa.bkg.model.transferobjects;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Data;
+import org.dcsa.bkg.model.enums.LocationType;
+import org.dcsa.bkg.model.enums.TransportPlanStage;
+import org.dcsa.core.events.model.Location;
+import org.dcsa.core.events.model.ModeOfTransport;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.time.OffsetDateTime;
+
+@Data
+public class ShipmentLocationTO {
+
+  @NotNull private Location location;
+
+  @Size(max = 250)
+  private String displayedName;
+
+  @NotNull
+  private LocationType locationType;
+
+  @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ssZ")
+  private OffsetDateTime eventDateTime;
```

Comment:
Ditto regarding the `@JsonFormat`. Also, I think this should be `LocalDate` but confirming that with Henrik.